### PR TITLE
Add write up for using cluster manager in baremetal environment

### PR DESCRIPTION
**Pull request by mapuri**
with `clusterctl discover` command the workflow is a bit more simplified.

/cc @jainvipin @rkharya PTAL

@rkharya if you get a chance please try this out and let me know if something doesn't work for you

### CODE REVIEW

**Comment by rkharya** [comment]
sure thanks, will get back with comment, if any.

**Comment by vvb** [comment]
@mapuri ..

**Comment by vvb** [comment]
@mapuri 
I had to manually start collins (I am on a copy of ansible that is 1 week old)
```
TASK [contiv_cluster : copy shell script for starting collins] *****************
ok: [node1]

TASK [contiv_cluster : copy systemd units for collins] *************************
ok: [node1]

TASK [contiv_cluster : check for collins image] ********************************
changed: [node1]

TASK [contiv_cluster : pull collins container image] ***************************
skipping: [node1]

TASK [contiv_cluster : start collins] ******************************************
changed: [node1]

```
[admin@ucsb-blade1 contiv]$ docker ps
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
[admin@ucsb-blade1 contiv]$

[admin@ucsb-blade1 contiv]$ docker images
REPOSITORY          TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
contiv/collins      02_25_2016          06c9f0b11539        4 weeks ago         659.8 MB
```

**Comment by mapuri** [comment]
@vvb you likely hitting #35 . Basically if collins get restarted or stopped as part of provisioning it affects cluetr manager which is ofcourse not desirable.

this shall be fixed when I fix #54 in next few days.

**Comment by mapuri** [comment]
@vvb  to avoid hitting #35 for the time being you can skip adding control host (i.e. the machine running cluster manager) to the cluster i.e. don't run `clusterctl discover` for that host. This is just short term thoguh

File: management/baremetal.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+##Preparing baremetal or personal VM environments for Cluster Management
+
+This document goes through the steps to prepare a baremetal or personal VM setup for cluster management.
+
+**Note:**
+- Unless explicitly mentioned all the steps below are done by logging into the same host. It is referred as *control host* below.
+- Right now we test cluster manager on Centos7.2. More OS variations shall be added in future.
+
```
Comment (rkharya) [inline management/baremetal.md]: @mapuri Can i try this out on a bare metal node running RHEL 7.2 stock os? Or this is in pipeline for next drop?

Comment (mapuri) [inline management/baremetal.md]: I think RHEL 7.2 shall work.

**Comment by mapuri** [comment]
I will merge this now assuming there are no doc related comments. Let's create separate issues for things that we hit following this.

**Pull request merged**